Sources/ProductHunt/SwiftUI/ProductHuntVotes.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  ProductHuntVotes.swift
+//  ProductHunt
+//  
+//
+//  Created by Julien Lacroix on 05/11/2020.
+//
+
+/// We want ProductHuntVotes available for iOS, tvOS, and watchOS.
+#if canImport(UIKit)
+import Foundation
+
+
+
+//MARK: - Class
+@available(iOS 14, *)
+class ProductHuntVotes: ObservableObject {
```

Comment:
Maybe it could be more consistent to name it `PHVotes` with  a `value` property, no?

Reply:
Done. You can check the new commit: "Clean code" - I modified the name of this class and the name of this property